backend/index.js

```diff
@@ -1,7 +1,48 @@
 require('dotenv').config();
 const http = require('http');
-
+const cronJob = require('./utility/cronJob');
 const cluster = require('./helpers/cluster');
+const joinUs = require('./app/models/joinUs');
+const sendEmail = require('./utility/sendEmail');
+const { JoinUsCronJobMailTemplate } = require('./utility/emailTemplates');
+const Admin = require('./app/models/Admin');
+
+// Running Cronjob for 2 months - 0 0 2 * *
+cronJob('* * * * 2 *', async () => {
```

Comment:
Let's keep it like `0 0 2 * *` so that it will run at 12am of every 2months 
\cc: https://stackoverflow.com/questions/26702952/how-setup-node-schedule-every-month